nexus/src/authz/omicron.polar

```diff
@@ -249,6 +251,10 @@ resource SiloUser {
 has_relation(silo: Silo, "parent_silo", user: SiloUser)
 	if user.silo = silo;
 
+# authenticated actors have all permissions on themselves
+has_permission(actor: AuthenticatedActor, _perm: String, silo_user: SiloUser)
+    if actor.equals_silo_user(silo_user);
```

Comment:
Is there any reason not to just do something like `actor.id = silo_user.id`?

Reply:
Yeah, we can't make UUIDs into Polar classes, since we don't own the crate.

Reply:
The other problem here is that there's no `id` attribute on `silo_user`.  We could add one, but this type is generated by the `authz_resource!` macro so either we'd have to make that configurable or everything would get a new `id` attribute.  I decided this isn't worth spending more time on.